### ISSUE TITLE
Separate the main package into its own `cmd/tandem`

### ISSUE THREAD

**Issue by shayne** [rosszurowski/tandem#5]
I'd love to use tandem's process manager in an existing Go project. We use Go itself to set up an environment, and then we manage the processes ourselves. It'd be great to swap in an API from tandem and benefit from all the features you've built.

**Comment by rosszurowski** [comment]
Good idea! I've wanted this myself. I'm not quite sure which bits will need to be exposed, but I've started the work in #6. I'm assuming exposing the `ProcessManager` and its `Run` command is enough for what you want? Might be fine to start there and add more as needed.

**Comment by shayne** [comment]
That sounds great. `ProcessManager` and `Run` would work!

**Comment by rosszurowski** [comment]
Added! Let me know if you need any tweaks to make things work!
